src/d3a/models/market/__init__.py

```diff
@@ -78,6 +80,13 @@ def __init__(self, time_slot=None, bc=None, notification_listener=None, readonly
             if transfer_fees is not None else 0
         self.transfer_fee_const = transfer_fees.transfer_fee_const \
             if transfer_fees is not None else 0
+
+        self.fee_class = GridFees
+        self.grid_fee_value = self.transfer_fee_ratio
+        if self.transfer_fee_const > 0:
+            self.fee_class = ConstantGridFees
+            self.grid_fee_value = self.transfer_fee_const
```

Comment:
I guess it could be in the else clause rather then assigning and then overwriting if `transfer_fee_const >0`

Reply:
One additional proposal, how about adding the grid_fee_value as an construction parameter and only use it inside the fee_class?

Reply:
In order to summarize both comments, I agree it would be better to convert the grid fees from a class with only static methods to an instance, and store/manage the grid fee values in the constructor of this class. Changing this, thanks